### DISCUSSION TITLE
【求助】如何部署绳网

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [share121/inter-knot#547] [崩坏：星穹铁道]
### Raw Issue at https://github.com/share121/inter-knot/issues/542

我也想部署一个绳网，但不知如何部署

**Comment by Buer-Nahida** [comment]
暂时关闭

**Comment by TrafficLight6** [comment]
我已经私信作者了，他说会把部署教程补上

**Comment by Buer-Nahida** [comment]
Github page部署成功了，但Cloudflare没部署上

**Comment by Buer-Nahida** [comment]
见https://buer-nahida.github.io/inter-knot

**Reply by TrafficLight6** [reply]
为什么明明安装了油猴插件，但是页面还是叫我安装

**Reply by lllliufeng** [reply]
> 为什么明明安装了油猴插件，但是页面还是叫我安装

那个插件只适用share121.github.io（这个绳网）吧

**Reply by TrafficLight6** [reply]
没事了，自己改了，可以用了，但数据还是_share121的discussion

**Comment by Buer-Nahida** [comment]
但帖子评论还没搭好，慢慢折腾吧

**Comment by TrafficLight6** [comment]
如何在github page部署？是直接来到自己的github page 仓库就行了吗？

**Comment by Buer-Nahida** [comment]
额，现在我用Cloudflare一键部署了，不用Github page。

之前的版本构建步骤是先Fork，再把`.github/workflows/main.yml`里的东西复制后删除这个文件再到Actions里面建个`.github/workflows/main.yml`，然后到Settings里的Pages分区设置成这样
![image](https://github.com/user-attachments/assets/daefd42a-7c3c-43b4-9b5e-ff47b5b2ab51)
然后就行了

**Comment by Buer-Nahida** [comment]
现在我又换成vercel了
